### ISSUE TITLE
[BUG] (UndefinedFunctionError) function :ping_erlang.__info__/1 is undefined or private

### ISSUE THREAD

**Issue by Mirk32** [aesmail/kaffy#299]
**Versions Used**
Kaffy: 0.10.0
Phoenix: 1.7
Elixir: 1.15.5-otp-26
Erlang: 26.0.2

On admin page loading get following error - (UndefinedFunctionError) function :ping_erlang.__info__/1 is undefined or private

Fixed locally by manual patching the method
deps/kaffy/lib/kaffy/utils.ex - get_schemas method with following code - 

```
  defp get_schemas(mods) do
    Enum.filter(mods, fn m ->
      if m == :ping_erlang do
        false
      else
        functions = m.__info__(:functions)
        Keyword.has_key?(functions, :__schema__) && Map.has_key?(m.__struct__, :__meta__)
      end
    end)
  end
```

Admin page should be displayed

**Screenshots**
<img width="1732" alt="image" src="https://github.com/aesmail/kaffy/assets/10832890/ed7dbe01-6976-40df-807e-7b4c4f322d94">

**Comment by aesmail** [comment]
@Mirk32 thank you for reporting the issue. Just released Kaffy v0.10.1 with a fix for the issue.

**Comment by aesmail** [comment]
acted too quickly.
the fix made Kaffy ignore all admin modules.
already released v0.10.2 to revert the change.
will look into this further.
